mainframer.sh

```diff
@@ -96,10 +96,12 @@ function syncBeforeRemoteCommand {
 function executeRemoteCommand {
 	echo "Executing command on remote machine…"
 	echo ""
+	echo "$REMOTE_COMMAND"
```

Comment:
This should be done on remote machine to evaluate possible env vars inside string correctly.